src/Wallabag/ApiBundle/Controller/UserRestController.php

```diff
@@ -97,29 +98,38 @@ public function putUserAction(Request $request)
                 ->setStatusCode(JsonResponse::HTTP_BAD_REQUEST);
         }
 
+        // create a default client
+        $client = new Client($user);
+        $client->setName('Default client');
```

Comment:
Instead of putting `Default client`, we could require the app to provide a name in the request.